src/components/AsideHeader/components/AllPagesPanel/AllPagesListItem/AllPagesListItem.module.scss

```
@@ -0,0 +1,29 @@
+@use '../../../../variables';
+@use '../../../../../../styles/mixins';
+
+.#{variables.$ns}all-pages-list-item {
+    @include mixins.accessibility-button;
+
+    display: flex;
+    align-items: center;
+
+    width: 100%;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)